blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeGraphVisitor.java

```diff
@@ -274,9 +274,13 @@ public void parallelStart(@Nonnull FlowNode parallelStartNode, @Nonnull FlowNode
                     status = new NodeRunStatus(genericStatus);
                 }
             }else{
-                times = new TimingInfo(TimingAction.getStartTime(branchStartNode)+System.currentTimeMillis(),
+                long startTime = System.currentTimeMillis();
```

Comment:
That's reasonable.  I think there's a problem here if start time is null but end time isn't -- but this fixes the B.O. logic at least.